src/java/fp/v12/FootPrints12Interface.java

```diff
@@ -0,0 +1,74 @@
+package fp.v12;
+
+/**
+ *  Describes the services offered by
+ *  FootPrint version 12
+ */
+public interface FootPrints12Interface {
```

Comment:
(J) as you probably realize this should be `ISplints` ... or an `ISplints` mapping to the API of FP12. Like what's an item? You can keep `FootPrints12Interface` for now then we need also `ISplints` here and an adapter implementing both.